### PR TITLE
Redirecting dialog and authentication message

### PR DESCRIPTION
**Pull request by uahmed-assecor**
Tasks completed:

[Loading animation or message when redirecting to AuthApp ](https://git.datev.de/mandat/backlog/-/issues/358)
[Add link to "not logged in" message box
](https://git.datev.de/mandat/backlog/-/issues/361)

### CODE REVIEW

File: libs/components/src/LoadingDialog.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<script setup lang="ts">
+
+import {ref, watch} from "vue";
+
+const isVisible = ref(false);
+
+const props = defineProps<{ isVisible: boolean }>();
+
+watch(()=> props.isVisible,()=> {
+  isVisible.value = props.isVisible;
```
Comment (cillies-finatix) [inline libs/components/src/LoadingDialog.vue]: This should not be needed. Just use the prop value in the template.

Remove `const isVisible = ref(false);`. You can then still use the template variable "isVisible" from the props.

File: libs/components/src/LoadingDialog.vue [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <Dialog v-model:visible="isVisible" modal header="Redirecting to the AuthApp" center style="width: 50%; margin-top: 2rem; display: block; margin-left: auto; margin-right: auto">
```
Comment (cillies-finatix) [inline libs/components/src/LoadingDialog.vue]: Get rid of the inline-styles, use primeflex :)

**Review by banxtv — APPROVED** [review]
made a little text change in loadingdialog, other than that it looks good! :)

**Comment by uahmed-assecor** [comment]
Dialog box appears and redirect also occur in the Lisa App.

**Pull request merged**